index.html

```diff
@@ -6,17 +6,29 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link href="/assets/CSS/style.css" rel="stylesheet" />
-  <title>Portofolio</title>
+  <link href="/assets/CSS/desktop.css" rel="stylesheet" />
+  <title>Portfolio</title>
 </head>
 <body>
```

Comment:
- [ ] As outlined by the previous reviews, kindly **ensure the entire page is centered on the screen**. Ensure that the extreme left and right are always equal.

Reply:
I justify content and justify items to centre